backend/dataall/modules/dataset_sharing/services/share_processors/s3_access_point_process_share.py

```diff
@@ -179,21 +186,33 @@ def clean_up_share(
         -------
         True if share is cleaned-up successfully
         """
-
-        clean_up = S3ShareManager.delete_access_point(
+        clean_up_folder = cls(
+            session,
+            dataset,
+            share,
+            folder,
+            source_environment,
+            target_environment,
+            source_env_group,
+            env_group,
+            existing_shared_buckets
```

Comment:
Do we need `existing_shared_buckets` here? I think we can remove it from here and from the init right?

Reply:
I think this is the source of error 2 reported in the testing below! @anushka-singh

Reply:
Yes we do. Its being used in the function below.
This was always working, I removed it by mistake while fixing something else. I just had to add existing_shared_buckets: bool = False in class ProcessS3AccessPointShare to fix it. 
Tested it on my local and it works fine now.

Reply:
I think I formulated it wrong, we need the variable, but do we need to pass the variable to the cls in like 200?

Reply:
Sure
Removed it from line 200